packages/api/amplify_api_dart/lib/src/api_plugin_impl.dart

```diff
@@ -140,10 +167,7 @@ class AmplifyAPIDart extends APIPluginInterface with AWSDebuggable {
     String? apiName,
     APIAuthorizationType? authorizationMode,
   }) {
-    final endpoint = _apiConfig.getEndpoint(
```

Comment:
Q: Can you remove AWSApiPluginConfigHelpers.getEndpoint() in this PR? Is it used anywhere?